exporter/loadbalancingexporter/testdata/config.yaml

```diff
@@ -27,3 +27,28 @@ loadbalancing/3:
     dns:
       hostname: service-1
       port: 55690
+loadbalancing/4:
+  protocol:
+    otlp:
+  resolver:
+    dns:
+      hostname: service-1
+      port: 55690
+  routing_key: traceID
+loadbalancing/5:
+  protocol:
+    otlp:
+  resolver:
+    dns:
+      hostname: service-1
+      port: 55690
+  routing_key: service
+loadbalancing/6:
+  protocol:
+    otlp:
+  resolver:
+    dns:
+      hostname: service-1
+      port: 55690
+  routing_key: resource
+  resource_keys: ["resource.attribute", "service.name"]
```

Comment:
Can you use a real attribute name, perhaps one from the semconv, instead of `resource.attribute`?